plugins/whatsapp.js

```
@@ -373,59 +416,59 @@
-		if (!message.reply_message.video) return message.send('_Reply A Video_');
-		media = await message.download();
-		return await message.send(media, { ptv: true });
-	}
+  {
+    pattern: 'ptv',
+    public: true,
+    desc: 'Convert video to pvt video note',
+    type: 'whatsapp',
+  },
+  async (message) => {
+    let media;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'media' is only assigned here.


(lint/style/useConst)